resources/v2/11_student_subjects.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE student_subjects 
+(
+     id serial NOT NULL PRIMARY KEY,
+     student_id integer NOT NULL,
+     subject_id integer NOT NULL,
+     created_at timestamp WITH time zone DEFAULT NOW(),
+     updated_at timestamp WITH time zone,
+     deleted_at timestamp WITH time zone,
+     CONSTRAINT student_subjects_student_id FOREIGN KEY
+     (student_id) REFERENCES students (id),
+     CONSTRAINT student_subjects_subject_id FOREIGN KEY
+     (subject_id) REFERENCES subjects (id)
```

Comment:
let's add a composite unique index on student_id and subject_id so that duplicates for them is not allowed. 

For example

student_id, subject__id
1,1 -> valid
1,2 -> valid
2,1 -> valid
2,2 -> valid
2,2 -> invalid (since it is duplicate, hence you need to unique on a combination of student_id, subject_id